src/mds/cephfs_features.h

```diff
@@ -45,7 +45,8 @@ namespace ceph {
 #define CEPHFS_FEATURE_ALTERNATE_NAME       15
 #define CEPHFS_FEATURE_NOTIFY_SESSION_STATE 16
 #define CEPHFS_FEATURE_OP_GETVXATTR         17
-#define CEPHFS_FEATURE_MAX                  17
+#define CEPHFS_FEATURE_MDS_AUTH_CAPS_CHECK  18
+#define CEPHFS_FEATURE_MAX                  18
```

Comment:
minor note: This will conflict with PR #48382

Reply:
Yeah, I will fix this once your PR gets merged.